flow/operations.py

```diff
@@ -104,6 +104,10 @@ class directives(object):
 
     Directives can for example be used to provide information about required resources
     such as the number of processes required for execution of parallelized operations.
+
+    In addition, you can use the `@directives(fork=True)` directive to enforce that a
+    particular operation is always executed within a subprocess and not within the
+    Python interpreter's process even if there are no other reasons that would prevent that.
     """
```

Comment:
```suggestion
+    .. note::
+
+        Setting `fork=False` will not prevent forking if there are other reasons for forking,
+        such as a deviating executable or a timeout argument.
    """
```

Reply:
"deviating executable" is not a word I recognize and couldn't find on Google. Maybe there's a better word? "An executable that creates a fork on its own"?

Reply:
I've updated my comment, better?

Reply:
fixed typo

Reply:
Using timeout as example instead of executable.